### PR TITLE
remove insomnia-config, move it inside insomnia

### PR DESCRIPTION
**Pull request by filfreire**
trying to move insomnia-config inside of insomnia package

pros: speeds up bootstrap
cons: we'll need to run generate manually when needed

### CODE REVIEW

**Review by jackkav — APPROVED** [review]
🎸

File: packages/insomnia/package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "test:watch": "jest --watch",
     "type-check": "tsc --noEmit --project tsconfig.build.json",
     "type-check:watch": "npm run type-check -- --watch",
-    "convert-svg": "npm_config_yes=true npx @svgr/cli@6.4.0 --no-index --config-file svgr.config.js --out-dir src/ui/components/assets/svgr src/ui/components/assets/"
+    "convert-svg": "npm_config_yes=true npx @svgr/cli@6.4.0 --no-index --config-file svgr.config.js --out-dir src/ui/components/assets/svgr src/ui/components/assets/",
+    "generate": "esr ./src/utils/config/generated/generate.ts"
```
Comment (gatzjames) [inline packages/insomnia/package.json]: Nitpick: could rename this to generate-config to be more clear

**Review by gatzjames — APPROVED** [review]
Looking awesome! 🚀

File: packages/insomnia/package.json (hunk 1 above)
Comment (filfreire) [reply packages/insomnia/package.json]: done

**Pull request merged**